dnf/cli/output.py

```diff
@@ -1518,12 +1518,10 @@ def historyListCmd(self, tids, reverse=False):
             name = _("Command line")
             real_cols = self.term.real_columns
             if real_cols is None:
-                name_width = (
-                        24 if not transactions
-                        else max([len(t.cmdline) for t in transactions])
-                        )
-            else:
-                name_width = real_cols - 55 if real_cols > 79 else 24
+                real_cols = dnf.cli.term._real_term_width(0)
```

Comment:
Please could you add a note why you try `_real_term_width(0)` (`stdin`)? On line 1519 it tries `real_cols = self.term.real_columns` => `dnf.cli.term._real_term_width(1)` `stdout`. I suggest that `less` provides length of `stdin` and not `stdout`. Am I correct?